tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
It can do both (`-c mac`), but AFAIK not at the same time. We should first check the overhead of the wrappers and only consider other solutions if it is really too high on huge files.